docs/en-US/component/table.md

```diff
@@ -7,6 +7,12 @@ lang: en-US
 
 Display multiple data with similar format. You can sort, filter, compare your data in a table.
 
+:::warning
+
+As of ^(2.12.0), customizing cell content via the default slot of `<el-table-column>` is deprecated. Please use the [cell slot](#custom-column-template-with-the-cell-slot) instead.
```

Comment:
Overall, we just added a new configuration, and there may be no problem in releasing it directly in the next patch version.

Reply:
Do you want me to remove it?

Reply:
Because warning messages will be prompted during development.

For the component like Table, I think users may not be able to accept such changes in the 2.11.2 version

Reply:
Well, what you said makes sense.